unit tests of the ./pkg/volume are failing

On a clean Ubuntu dev environment I get the following error when I executed the `make test WHAT=./pkg/volume`

```
+++ [0604 10:05:38] Running tests without code coverage
--- FAIL: TestGetMetricsBlock (0.11s)
  metrics_block_test.go:96: Unexpected error when calling GetMetrics: unable to open device "/dev/cdrom": open /dev/cdrom: no medium found
```
  
I think that the tests should not fail and should handle the case where there is a cdrom device but there is not an actual CD media inserted.

I got feedback from @serathius and this is also happing on his environment with the following error:

```
--- FAIL: TestGetMetricsBlock (0.00s)
    metrics_block_test.go:92: Unexpected error when calling GetMetrics: unable to open device "/dev/dm-0": open /dev/dm-0: permission denied
```
    
I am coping also his comment from the slack that may help: 
_Looks like it's not a problem with cdrom, but some dependence on device. I don't think it should try try to access at all._
    
This is my very first issue, I am new to this, any help is welcome.
I am really looking forward to starting coding but I am overwhelmed with the amount of information that I need to digest.

/cc serathius
/sig instrumentation

--- a/pkg/volume/metrics_block_test.go
+++ b/pkg/volume/metrics_block_test.go
@@ -17,9 +17,6 @@ limitations under the License.
 package volume_test
 
 import (
-	"io/fs"
-	"os"
-	"runtime"
 	"testing"
 
 	. "k8s.io/kubernetes/pkg/volume"
@@ -44,55 +41,5 @@ func TestGetMetricsBlockInvalid(t *testing.T) {
 	}
 	if err == nil {
 		t.Errorf("Expected error when calling GetMetrics on incorrectly initialized MetricsBlock, actual nil")
-	}
-}
-
-func TestGetMetricsBlock(t *testing.T) {
-	// FIXME: this test is Linux specific
-	if runtime.GOOS == "windows" {
-		t.Skip("Block device detection is Linux specific, no Windows support")
-	}
-
-	// find a block device
-	// get all available block devices
-	//  - ls /sys/block
-	devices, err := os.ReadDir("/dev")
-	if err != nil {
-		t.Skipf("Could not read devices from /dev: %v", err)
-	} else if len(devices) == 0 {
-		t.Skip("No devices found")
-	}
-
-	// for each device, check if it is available in /dev
-	devNode := ""
-	var stat fs.FileInfo
-	for _, device := range devices {
-		// if the device exists, use it, return
-		devNode = "/dev/" + device.Name()
-		stat, err = os.Stat(devNode)
-		if err == nil {
-			if stat.Mode().Type() == fs.ModeDevice {
-				break
-			}
-		}
-		// set to an empty string, so we can do validation of the last
-		// device too
-		devNode = ""
-	}
-
-	// if no devices are found, or none exists in /dev, skip this part
-	if devNode == "" {
-		t.Skip("Could not find a block device under /dev")
-	}
-
-	// when we get here, devNode points to an existing block device
-	metrics := NewMetricsBlock(devNode)
-	actual, err := metrics.GetMetrics()
-	if err != nil {
-		t.Errorf("Unexpected error when calling GetMetrics: %v", err)
-	}
-
-	if a := actual.Capacity.Value(); a <= 0 {
-		t.Errorf("Expected Capacity %d to be greater than 0.", a)
 	}
 }

